src/python/tools/Report_InvalidUrls_FindHttpURLs.csv

```diff
@@ -0,0 +1,18 @@
+url	relativeFilepath
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\CommonCpp.props
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\NativeBridge\NativeBridge.vcxproj
+http://schemas.microsoft.com/developer/msbuild/2003	.\src\python\nimbusml.pyproj
+http://www.nvidia.com/Drivers	.\src\DotNetBridge\MessageValidator.cs
+http://www.jmlr.org/papers/volume14/shalev-	.\src\python\docs\docstrings\FastLinearBinaryClassifier.txt
```

Comment:
>http://www.jmlr.org/papers/volume14/shalev- [](start = 0, length = 43)

url is different in source file

Reply:
Indeed, the correct link is `http://www.jmlr.org/papers/volume14/shalev-shwartz13a/shalev-shwartz13a.pdf`. However, this website doesn't support HTTPS. This link and the one above were cut into two lines, hence why FindHttpURLs.py didn't pick it up as a non-alterable HTTP URL.